test/integration/AgentTest.js

```diff
@@ -508,7 +508,7 @@ test('Agent 9: Clear Path', t => {
   }));
 });
 
-/*test.only('Agent 10: Ride Rails', t => {
+/*test('Agent 10: Ride Rails', t => {
```

Comment:
Changes to this commented-out test? 😛

Reply:
Well I didn't want to forget it when someone removes the comments later, provided that test -ever- works 😭